ctk/features/for.feature

```diff
@@ -0,0 +1,35 @@
+Feature: For Task
+  As an implementer of the workflow DSL
+  I want to ensure that for tasks can be executed within the workflow
+  So that my implementation conforms to the expected behavior
+
+  # Tests for tasks
+  # Tests named iteration item (i.e.: `color`)
+  # Tests default iteration index (`index`)
+  Scenario: For Task
+    Given a workflow with definition:
+    """yaml
+    document:
+      dsl: 1.0.0-alpha1
+      namespace: default
+      name: for
+    do:
+      forEachColor:
+        for:
+          each: color
+          in: '.colors'
+        do:
```

Comment:
Can be several "dos" being performed for every iteration of the loop?
If so, how can the second "do" refer to the result of the first "do"?

Reply:
No, you can only perform a single do. However it can be a composite task made out of multiple subtasks.

Reply:
Does that resolve the conversation?

Reply:
yes, for some reason Im unable to resolve the comment ;)